lib/src/switch/Switch.stories.tsx

```diff
@@ -0,0 +1,154 @@
+import React from "react";
+import { userEvent, within } from "@storybook/testing-library";
+import DxcSwitch from "./Switch";
+import { BackgroundColorProvider } from "../BackgroundColorContext";
+import Title from "../../.storybook/components/Title";
+import ExampleContainer from "../../.storybook/components/ExampleContainer";
+import DarkContainer from "../../.storybook/components/DarkSection";
+
+export default {
+  title: "Switch",
+  component: DxcSwitch,
+};
+
+export const Chromatic = () => (
+  <>
+    <ExampleContainer>
+      <Title title="With label before" theme="light" level={4} />
+      <DxcSwitch label="Switch" />
+    </ExampleContainer>
+    <ExampleContainer>
+      <Title title="With label after" theme="light" level={4} />
+      <DxcSwitch label="Switch" labelPosition="after" />
+    </ExampleContainer>
+    <ExampleContainer>
+      <Title title="Without label" theme="light" level={4} />
+      <DxcSwitch />
+    </ExampleContainer>
+    <ExampleContainer>
+      <Title title="With large label" theme="light" level={4} />
+      <DxcSwitch label="Very long long long label or huge as you want to call it" size="medium" />
+    </ExampleContainer>
+    <ExampleContainer>
+      <Title title="With long word" theme="light" level={4} />
+      <DxcSwitch
+        label="Large texttttttttttttttttttttttttttttttttttttttttttttttttttttttttttttt"
+        labelPosition="after"
+        size="medium"
+      />
+    </ExampleContainer>
+    <ExampleContainer>
```

Comment:
We should test the ellipsis for medium size also. We can add these ellipsis tests in the sizes test section as in the button component, for example.

Reply:
I agree!